implementation/src/main/java/io/smallrye/config/ConfigValueConfigSource.java

```diff
@@ -401,14 +407,18 @@ private void load0(LineReader lr) throws IOException {
                 }
                 String key = loadConvert(lr.lineBuf, 0, keyLen, convtBuf);
                 String value = loadConvert(lr.lineBuf, valueStart, limit - valueStart, convtBuf);
-                put(key, ConfigValue.builder()
+                ConfigValue oldConfigValue = put(key, ConfigValue.builder()
                         .withName(key)
                         .withValue(value)
                         .withRawValue(value)
                         .withConfigSourceName(configSourceName)
                         .withConfigSourceOrdinal(configSourceOrdinal)
                         .withLineNumber(lr.lineNumber)
                         .build());
+                if (oldConfigValue != null) {
+                    log.warnv("duplicate keys found for : {0}, source name : {1}", oldConfigValue.getName(),
```

Comment:
Can you please add a new method in https://github.com/smallrye/smallrye-config/blob/029c88ea3f957417b04231748b6d2431caed2959/implementation/src/main/java/io/smallrye/config/_private/ConfigLogging.java#L13 for the log message?

Reply:
Ok, Is it ok this way? 

https://github.com/fugerit-org/smallrye-config/commit/c562fb9b71c9ca2f93c7226ab73adc3f277429e9

Reply:
Yes :)